TestApp/android/gradle.properties

```diff
@@ -16,5 +16,3 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-
-android.useDeprecatedNdk=true
```

Comment:
@roland-chernov-akvelon Do we still need this or it's safe to get rid of it?
@guperrot Any comments?

Reply:
There were a lot of changes generated by the update utility. We kept only necessary and reverted this file back. Please let me know if we can remove this line.

Reply:
See [RN v0.57.3 release notes](https://github.com/facebook/react-native/commit/6117a6c7205c969f93d39ba02e0583881572d5fa#diff-975c8e77f36db951fb304aa3d843b2fd)

> bump Android NDK to r17c (b7d0594 by @dulmandakh)

and [diff which removes `android.useDeprecatedNdk=true`](https://github.com/facebook/react-native/commit/6117a6c7205c969f93d39ba02e0583881572d5fa#diff-975c8e77f36db951fb304aa3d843b2fd).

Reply:
Thank you, removed.

Reply:
We don't support NDK in RN SDK yet so it should be safe to remove this line.